argoverse/data_loading/object_label_record.py

```diff
@@ -207,6 +238,56 @@ def draw_rect(selected_corners: np.array, color: Tuple[int, int, int]) -> None:
         return img
 
 
+def add_text_cv2(img: np.ndarray, text: str, x: int, y: int, color: Tuple[int,int,int], thickness: int = 3) -> None:
+    """Add text to image using OpenCV. Color should be RGB order"""
```

Comment:
I think there's an anti-aliasing option in `cv`. Might make the text look a bit nicer -- just a thought.

Reply:
Added, thanks for the suggestion